FlaxEditor/Surface/ContextMenu/VisjectCM.cs

```diff
@@ -19,18 +19,19 @@ public class VisjectCM : ContextMenuBase
         private bool _waitingForInput;
         private VisjectCMGroup _surfaceParametersGroup;
         private Panel _panel1;
-        private VerticalPanel _panel2;
+        private VerticalPanel _groupsPanel;
         private Func<List<SurfaceParameter>> _parametersGetter;
+        private Elements.Box _selectedBox;
 
         /// <summary>
         /// The selected item
         /// </summary>
         public VisjectCMItem SelectedItem;
-        
+
         /// <summary>
         /// Event fired when any item in this popup menu gets clicked.
         /// </summary>
-        public event Action<VisjectCMItem> OnItemClicked;
+        public event Action<VisjectCMItem, Elements.Box> OnItemClicked;
```

Comment:
Please add a delegate to provide documentation for event parameters (the added box parameter is not so obvious).